### ISSUE TITLE
Mixxx unresponsive/very slow when analysing even on a fast computer

### ISSUE THREAD

**Issue by veeroohre** [mixxxdj/mixxx#12857]
### Bug Description

I have a quite fast CPU (AMD Ryzen 7 7800X3D). Nevertheless the program freezes and is very slow when analysing my library.
All other programms/Windows works normal. My library is on a NAS, so the files are being accessed via SMB.

Maybe it's because of the "network card load"? If true, I don't see a reason for this behaviour because the analysing runs in a threaded process, right?

![Screenshot 2024-02-21 141530](https://github.com/mixxxdj/mixxx/assets/8831160/06100294-4149-477f-a0cf-1f5274038df7)

### Version

2.4.0

### OS

Windows 11

**Comment by daschuer** [comment]
Yes, the analysis is multi threaded. However the GUI shall not freeze longer.

**Comment by veeroohre** [comment]
so it's a bug/problem? Can I test it somehow or provide more data/info?

**Comment by veeroohre** [comment]
It's only Mixxx. As I wrote there is a lot of network access due to my library being on a NAS. Nevertheless Mixxx freezes without doing something. Windows is showing "no response" on the app in the task bar.

**Comment by Swiftb0y** [comment]
right, so the behavior can differ between applications. It's very plausible that this is only a problem for mixxx. Can you try analyzing some local files instead and see if that has the same problem? Than we can rule out whether the fact that your files aren't local is at fault.

**Comment by veeroohre** [comment]
Did an analyse run with local files. Huge difference. CPU load is much higher (which is not surprising) but app is much more responsive.

Made two videos. Maybe it's clearer then.

Local files:
https://github.com/mixxxdj/mixxx/assets/8831160/6f371cea-bad1-47eb-b810-6ebd14f9b882

Samba share files (when the mouse cursors does not move in the video I made a left click):
https://github.com/mixxxdj/mixxx/assets/8831160/4642d9e5-7ccd-4808-b783-8a70cfc9dc68

**Comment by cyberchrime** [comment]
I tried to reproduce this with Mixxx 2.4 on Fedora Linux 39 on my Laptop (i7-4600U).

I mounted a samba share which ran inside a virtual machine (Ubuntu) on another computer. My laptop and the other computer are only connected with WiFi 5, so I assume there exists some latency to access the share.

Unfortunately, I was unable to fully reproduce this. Mixxx' GUI became a bit laggy, but I was still able to load songs into the decks and run playback while running analysis.

**Comment by veeroohre** [comment]
Thank you for your feedback. Is there any Windows user who could test, too?
